crates/node/core/src/args/bitfinity_args.rs

```diff
@@ -56,6 +56,10 @@ pub struct BitfinityImportArgs {
     /// Root key for the IC network
     #[arg(long, value_name = "IC_ROOT_KEY", default_value = IC_MAINNET_KEY)]
     pub ic_root_key: String,
+
+    /// A flag to stop importing blocks when the node is in staging mode
+    #[arg(long, default_value = "false")]
+    pub stop_importing_when_staging: bool,
```

Comment:
The logic is wrong. It is not about `staging` but about `enabled`.
The flag should be `check_evm_state_before_importing` then, if `true`, the blocks will be imported only if the evm state is `Enabled`